doc/src/base/statistics.md

```diff
@@ -0,0 +1,8 @@
+# Statistics
```

Comment:
Given that this page is very short, we could also put these under the Mathematics section, which already has multiple sections. But it's already quite long.

Reply:
We can leave it like this, and it is easy to change later on.